index.html

```diff
@@ -5007,13 +4983,259 @@ <h3 id="looping-AudioBufferSourceNode">
+            "#widl-AudioBufferSourceNode-start-void-double-when-double-offset-double-duration">
+              <code>start()</code></a> if supplied, else 0.
+            </li>
+            <li>Let <code>stop</code> correspond to the sum of the
+            <code>when</code> and <code>duration</code> parameters to
+              <a href="#widl-AudioBufferSourceNode-start-void-double-when-double-offset-double-duration">
+              <code>start()</code></a> if supplied, or the <code>when</code>
+              parameter to <code>stop()</code>, otherwise <em>Infinity</em>.
+            </li>
+            <li>Let <code>buffer</code> be the <a>AudioBuffer</a> employed by
+            the node.
+            </li>
+            <li>Let <code>offset</code> correspond to the <code>offset</code>
+            parameter to <a href=
+            "#widl-AudioBufferSourceNode-start-void-double-when-double-offset-double-duration">
+              <code>start()</code></a> if supplied, else 0. This is the
+              <a>playhead position</a> in the buffer at which playback will
+              begin.
+            </li>
+            <li>Let <code>loop</code>, <code>actualLoopStart</code> and <code>
+              actualLoopEnd</code> be the loop-related attributes of the node,
+              as discussed in the preceding section on looping.
+            </li>
+            <li>Let the function <code>playbackSignal(position)</code> be the
+            <em>playback signal function</em> for <code>buffer</code>, which is
+            a function that maps from a <a>playhead position</a> to a set of
+            output signal values, one for each output channel. If
+            <code>position</code> corresponds to the location of an exact
+            sample frame in the buffer, this function returns that frame.
+            Otherwise, its return value is determined by a UA-supplied
+            algorithm that interpolates between sample frames in the
+            neighborhood of <code>position</code>. If <code>position</code> is
+            greater than or equal to <code>actualLoopEnd</code> and there is no
+            subsequent sample frame in <code>buffer</code>, then the next
+            sample frame should be taken as the playback signal at
+            <code>actualLoopStart</code> for interpolation purposes.
+            </li>
+            <li>Let <code>bufferTime</code> equal -1. This is the <em>playhead
+            position</em> within <code>buffer</code> of the next output sample
+            frame. The initial value -1 indicates that this position has not
+            yet been determined.
+            </li>
+            <li>For each block of audio to be rendered:
+              <ol>
+                <li>Let <code>currentTime</code> be the value of
+                <code>currentTime</code> attribute of the node's
+                <a>AudioContext</a>.
+                </li>
+                <li>Let <code>contextRate</code> be the value of the
```

Comment:
Why give the sample rate the new name `contextRate`?  We don't refer to the sample rate of the buffer anymore, so there's no chance of confusion in this algorithm.